composer.json

```diff
@@ -9,20 +9,20 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.2 || ^8.0",
         "knplabs/knp-menu-bundle": "^3",
         "psr/simple-cache": "^1.0",
         "rakit/validation": "^1.2",
         "symfony/cache": "^3.4 || ^4.4",
         "symfony/framework-bundle": "^3.4 || ^4.4",
         "symfony/routing": "^3.4 || ^4.4",
-        "twig/twig": "^2",
+        "twig/twig": "^2.7 || ^3",
```

Comment:
De Htmldev Bundle is wel DE bundle met een hele hoop Twig. Weet je zeker dat alles compatible is met Twig 3?

Reply:
* localhost/htmldev werkt
* lint:twig geeft geen errors
* checks in deze PR draaien met Twig 3 en zijn :green_apple:
* Verschil tussen Twig 2 en 3 is voornamelijk de functie/namespace transitie

Reply:
Zou bovenstaande genoeg moeten zijn?

Reply:
Zou bovenstaande genoeg moeten zijn?

Reply:
Prima 👍🏻